packages/core/admin/package.json

```diff
@@ -62,7 +62,7 @@
     "browserslist-to-esbuild": "1.2.0",
     "chalk": "^4.1.1",
     "chokidar": "^3.5.1",
-    "codemirror": "^5.65.11",
+    "codemirror5": "npm:codemirror@^5.65.11",
```

Comment:
Can you explain to me why this solution is necessary? Is it trying to fetch v6 of `codemirror`?

Reply:
I think the new JSONInput in the DS requires codemirror@6

Reply:
But we import our own version from there, unsure why yarn creates the conflict. Maybe it's just worth updating it here?

Reply:
I'm going to see if we can update codemirror in the admin panel.

Reply:
Maybe create a follow-up ticket for now? I think it requires more than updating the package unfortunately.

Reply:
Sometimes yarn can handle multiple variants in workspaces satisfactorily. Still, after you release them, it becomes a single environment at the user environment level, and yarn attempts to treat it as a single variant. As a result, things break, and we need aliasing at the package manager level. (I guess...)

Reply:
And, unfortunately, getting rid of codemirror5 wouldn't be that minor efforts...